plugins/azure-sites-backend/src/service/router.ts

```diff
@@ -52,7 +65,33 @@ export async function createRouter(
     '/:subscription/:resourceGroup/:name/start',
     async (request, response) => {
       const { subscription, resourceGroup, name } = request.params;
-      console.log('starting...');
+      const token = getBearerTokenFromAuthorizationHeader(
+        request.header('authorization'),
+      );
+      const entity = request.body.entity;
```

Comment:
it isn't secure to send the full entity as payload, as an attacker might manipulate the content of the entity.
An alternative approach could be to send only the entityRef of the entity and then `azure-site-backend` can retrieve the full entity through  `catalogApi.getEntityByRef()`

Reply:
This is changed to pass entityRef instead entity